web/components/issues/issue-layouts/properties/all-properties.tsx

```diff
@@ -240,7 +240,7 @@ export const IssueProperties: React.FC<IIssueProperties> = observer((props) => {
       {/* basic properties */}
       {/* state */}
       <WithDisplayPropertiesHOC displayProperties={displayProperties} displayPropertyKey="state">
-        <div className="h-5">
+        <div className="h-5 truncate">
```

Comment:
Can you add this logic inside the state drowdown itself? it will cater to where ever it is being used